tests/ecdhp256-test.c

```diff
@@ -89,8 +93,134 @@ int main()
 	ok = ok && compare_and_print(64, point_compressed, compressed7);
 
 
+	printf("%s\n", "Wycheproof tests ECDH: ");
+		
+	for (int i = 0 ; i< sizeof(w_vectors)/sizeof(ecdhp256_w_i); i++)
+	{
+		printf("ECDH Wycheproof Test %d ", i );
+
+		uint8_t* decompressedPoint = (uint8_t*) malloc (sizeof (uint8_t) * 64);
+		bool flagDecompress = Hacl_P256_decompression_not_compressed_form(w_vectors[i].publicKey, decompressedPoint);
+
+		uint64_t success = Hacl_P256_ecp256dh_r(result, decompressedPoint, w_vectors[i].privateKey);
+
+		uint64_t fDU = 0;
+		if (flagDecompress = true)
```

Comment:
- assignment instead of comparison? this condition always evaluates to true, wasn't this flagged as a warning from the C compiler?
- what does `fDU` mean? let's use meaningful names, especially if we expect people to look at this code
- even imagining `==` instead of `=` I still don't understand the logic, please explain with a comment